File: Models/Snack.cs
using System.ComponentModel.DataAnnotations;

namespace MidnightCityTheater.Models;

public class Snack
{
    [Key]
    public int IdSnack { get; set; }
    public List<Pipoca>? Pipocas { get; set; }
    public List<Bebida>? Bebidas { get; set; }
    public List<Doce>? Doces { get; set; }
    public decimal PrecoTotal { get; set; }
}

File: Views/FrontMidnightCityTheater/src/app/Snack.ts
import {Pipoca} from "./Pipoca";
import {Bebida} from "./Bebida";
import {Doce} from "./Doce";

export class Snack {
    idSnack: number = 0;
    pipocas: Array<Pipoca> = [];
    bebidas: Array<Bebida> = [];
    doces: Array<Doce> = [];
    precoTotal: number = 0;
}